package.json

```diff
@@ -40,6 +40,7 @@
     "isomorphic-fetch": "^2.2.1",
     "js-base64": "^2.4.8",
     "lodash": "^4.17.19",
+    "moment": "^2.27.0",
```

Comment:
No need for `moment`

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat